docs/vercel.json

```diff
@@ -1,13 +1,14 @@
 {
-  "version": 2,
-  "name": "primer-viewcomponents",
   "builds": [
     {
       "src": "package.json",
-      "use": "@now/static-build",
+      "use": "@vercel/static-build",
       "config": {"distDir": "public"}
```

Comment:
I didn't change this but FYI there's no way for us to deploy  _just_ the public folder because [Vercel requires all static content to be in a `public` folder](https://vercel.com/guides/upgrade-to-zero-configuration#frameworks-with-zero-configuration); which is why we're deploying the whole docs folder but ignoring everything but the public folder (e531b7e1cab0d22a326cf337c1ce4c26c354b2a0)